.github/workflows/build-gtk.yml

```diff
@@ -37,6 +37,10 @@ jobs:
       - name: Build MAUI
         run: |
             dotnet build Microsoft.Maui.Gtk.slnf
+      - name: Build ZXing.Net.Maui
+        run: |
+            dotnet build ./src/ZXing.Net.Maui/ZXing.Net.MAUI/ZXing.Net.MAUI.csproj --framework=net6.0-gtk
```

Comment:
Can't we add these to the slnf file?

Reply:
@webwarrior-ws

Reply:
I don't see a reason. We have 2 projects, not 10.

Reply:
You are right this way works too. I thought that way would be cleaner since the goal of the slnf file was to pack all packages necessary for gtk. what do you think? @knocte

Reply:
I agree with having one slnf as well BUT since the name of that slnf is Microsoft.Maui I don't like adding ZXing to it.
If you go with adding it  to slnf you'll need a name change.